### PR TITLE
[WIP] Azure DevOps

### PR DESCRIPTION
**Pull request by igorshmukler**

### CODE REVIEW

File: .vsts-ci.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+trigger:
+- master
```
Comment (mcollina) [inline .vsts-ci.yml]: what does this mean? Can you add a comment?

Comment (igorshmukler) [reply .vsts-ci.yml]: `trigger` specifies which branch[es] would trigger the CI run, as per https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts#trigger
we also run CI on PRs to `master`.

do you believe that this should go into the `md` file?

Comment (mcollina) [reply .vsts-ci.yml]: All branches should trigger a CI run.

Comment (igorshmukler) [reply .vsts-ci.yml]: Cool. Will adjust. Below is a link to the short document that I made about the setup:

https://docs.google.com/document/d/1GlxrpRfw7VClScYFQk5H_y5fUhA6kIuidbbx2I0cBio/edit?usp=sharing

**Comment by mcollina** [comment]
Superseded by https://github.com/nearform/node-clinic/pull/93

**Pull request closed without merging**